crates/api_crud/src/comment/update.rs

```diff
@@ -65,13 +65,19 @@ impl PerformCrud for EditComment {
     }
 
     if data.distinguished.is_some() {
-      // Verify that only a mod or admin can distinguish a comment
-      is_mod_or_admin(
+      let is_mod = CommunityView::is_mod(
         context.pool(),
         local_user_view.person.id,
         orig_comment.community.id,
       )
       .await?;
+      let is_admin = local_user_view.person.admin;
+      let is_local_community = orig_comment.community.local;
+      if !is_mod || (is_admin && !is_local_community) {
```

Comment:
This isn't the right place to do this IMO, its in the apub receives, which are allowing people to impersonate themselves as admins, and send admin actions to other instances. Its a larger apub receive security issue, not an issue with the crud API.

The check should be simple tho:

`allow_distinguish = is_mod(person.id)`...

Reply:
I also made the change in apub receiver. If I removed this, then admins could distinguish comments in remote communities, but that action wouldnt federate. Dont think thats expected behaviour at all.

Reply:
Its a similar issue to admins removing posts, or featuring posts... they should have full control over their instance. IE if they want to distinguish a comment, or feature on their instance, why not, but it shouldn't get federated. 

I *could* remove admins seeing those options from the UX, but most important is that the apub on other servers reject that action.

Reply:
Disagree, we also dont let admins feature or lock posts in remote communities. I guess we could add a feature "local distinguish" for admins, similar to "local featured". But that would be a separate feature.

Reply:
That sounds wrong but as long as no one complains. Removed the api and db changes to keep it simple.